src/mon/OSDMonitor.cc

```diff
@@ -8004,33 +8004,6 @@ int OSDMonitor::prepare_new_pool(string& name,
   if (name.length() == 0)
     return -EINVAL;
 
-  if (pg_num == 0) {
-    auto pg_num_from_mode =
-      [pg_num=g_conf().get_val<uint64_t>("osd_pool_default_pg_num")]
-      (const string& mode) {
-      return mode == "on" ? 1 : pg_num;
```

Comment:
Ah, I see – we always start from `1` if autoscaler is supposed to take care about a pool.

Reply:
Yes, that's right.